static/templates/partials/iframely-widget-card.tpl

```diff
@@ -10,7 +10,7 @@
 
             <!-- IF title -->
             <h4 class="media-heading">
-                <a href="{embed.url}" target="_blank" rel="nofollow" class="one-line">
+                <a href="{embed.meta.canonical}" target="_blank" rel="nofollow" class="one-line">
```

Comment:
It's the same as meta.canonical for everything I've tried on iframely's API, and never set by the open source self-hosted version.